packages/shared/src/UI/components/AssetPlayer/index.tsx

```diff
@@ -17,32 +27,12 @@ interface AssetPlayerProps
         loop?: boolean
         muted?: boolean
     }
+    erc721Token?: ERC721TokenQuery
     iconProps?: SvgIconProps
+    fallbackImage?: URL
+    setERC721TokenName?: (name: string) => void
 }
-const useStyles = makeStyles()((theme) => ({
-    errorPlaceholder: {
-        backgroundColor: theme.palette.background.default,
-        display: 'flex',
-        justifyContent: 'center',
-        alignItems: 'center',
-        borderRadius: 12,
-        width: '100%',
-    },
-    errorIcon: {
-        width: 36,
-        height: 36,
-    },
-    loadingPlaceholder: {
-        display: 'flex',
-        justifyContent: 'center',
-        alignItems: 'center',
-        width: '100%',
-    },
-    loadingIcon: {
-        width: 36,
-        height: 52,
-    },
-}))
```

Comment:
Why remove these？

Reply:
![image](https://user-images.githubusercontent.com/63733714/148211287-52cec26a-8e1d-41e9-92b0-74677920bb1d.png)


There're some NFT components that have very different UI.